### DISCUSSION TITLE
Namecheap DNS verification, credential information missing

### DISCUSSION THREAD

**Discussion by moracarlos** [go-acme/lego#1572] [Q&A] [answered]
### Welcome

- [X] Yes, I'm using a binary release within 2 latest releases.
- [X] Yes, I've searched similar issues on GitHub and didn't find any.
- [X] Yes, I've included all information below (version, config, etc).

### What did you expect to see?

Created certificate

### What did you see instead?

Hello, I'm trying to create a certificate using lego, and when I exec:

```
NAMECHEAP_API_USER=<user> \
NAMECHEAP_API_KEY=<key> \
sudo /opt/bitnami/letsencrypt/lego --dns namecheap --domains <domain> --email <email> --tls run
```
And I get:
```
namecheap: some credentials information are missing: NAMECHEAP_API_USER
,NAMECHEAP_API_KEY
```
### How do you use lego?

Binary

### Reproduction steps
```
NAMECHEAP_API_USER=<user> \
NAMECHEAP_API_KEY=<key> \
sudo /opt/bitnami/letsencrypt/lego --dns namecheap --domains <domain> --email <email> --tls run
```
### Version of lego

```console
4.6.0
```

### Logs

<details>

```
2022/01/28 02:21:54 namecheap: some credentials information are missing: NAMECHEAP_API_USER
,NAMECHEAP_API_KEY
```

</details>

### Go environment (if applicable)

<details>

```console
$ go version && go env
# paste output here
```

</details>

**Comment by ldez** [comment, marked as answer]
Hello,

your problem is related to the `sudo` command and the env vars.
The two things are not in the same process. 

You have to do that:
```
sudo \
NAMECHEAP_API_USER=<user> \
NAMECHEAP_API_KEY=<key> \
/opt/bitnami/letsencrypt/lego --dns namecheap --domains <domain> --email <email> --tls run
```

**Comment by moracarlos** [comment]
Thanks a lot! It worked
